src/trio/_subprocess.py

```diff
@@ -723,7 +723,7 @@ async def read_output(
     ) -> None:
         async with stream:
             async for chunk in stream:
-                chunks.append(chunk)
+                chunks.append(chunk)  # noqa: PERF402
```

Comment:
I feel like `PERF402` on variable passed in is incorrect and probably is a bug...

Reply:
Created https://github.com/astral-sh/ruff/issues/11316 for this